README.md

```diff
@@ -1,5 +1,122 @@
+
+### Mapbox
+You'll first have to login or create an account [here](https://account.mapbox.com/auth/signin/). Update "mapbox_access_token" with your default public token in [strings.xml](app/src/main/res/values/strings.xml).
+```
+    <string name="mapbox_access_token">PASTE YOUR PUBLIC TOKEN HERE</string>
+```
+ Finally generate a private token and set it up in [build.gradle](./build.gradle) under password:
+```
+   credentials {
+                // This should always be `mapbox` (not the real username).
+                username = 'mapbox'
+                //Use Mapbox's secret key
+                password = PASTE YOUR PRIVATE KEY HERE
+            }
+```
+
+### OpenWeather
+You'll first have to login or create an account [here](https://home.openweathermap.org/users/sign_in). Then under "API keys" create a new key and copy it. All you have to do is add a new file secrets.xml in [res/values](app/src/main/res/values/) with the following code:
+```
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <item name="open_weather_api_key" type="string">PASTE YOUR KEY HERE</item>
+</resources>
+```
+
+## Developement
+Drone3D was developed as part of the CS-306 course "Software developement project" at EPFL. We have followed agile development methods by adopting the scrum framework. You cane find our scrum board [here](https://github.com/Drone3D-Team/Drone3D/projects/1). We have divided the semester into sprints of one week each. For each sprint we assigned ourselves tasks. Once a task has been implemented, tested and merged into the main branch, it is moved to the corresponding "Done is sprint X" column.  At the end of the week, a presentation of the application's functionality takes place, as well as a review of the last sprint. We summarise the work done in each sprint in a sprint summary available in our [wiki](https://github.com/Drone3D-Team/Drone3D/wiki).
+
+We had a few imposed requirements for this app:
+
+### Correctness & Utility
+Drone3D lets users plan and execute mission to create 3D model of a area.
+
+### Split app model
+Our app uses firebase to let the users store and share their mission.
+
+### Sensor usage
+Every map in the app is zoomed on the user, based on his gps. Users can follow the live progression of the drone on the map when a mission is launched. At any time they can send an instruction so that the drone comes back to their location.
+
+### User support
+We use an email/password register and login system. Logged-in users have the possibility to store and share their mission. The app is still useable without being logged-in.
+
+### Local cache & offline mode 
+Users have the possibility to download parts on the map before entering the mission site in case there is little or no connection there. They can also store and share mapping mission locally, it'll be synchronised with the online database as soon as they regain connection.
```

Comment:
Typo and I think it's better like this
```suggestion
Users have the possibility to download parts on the map before entering the mission site in case there is little or no connection there. They can also store and share mapping missions locally. It'll be synchronized with the online database as soon as they regain connection.
```